.github/workflows/ci.yaml

```diff
@@ -4,20 +4,39 @@ on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: windows-latest
 
     steps:
-    - uses: actions/checkout@v1
-      
+    - uses: actions/checkout@v2
+    - run: |
+        git fetch --prune --unshallow origin HEAD
```

Comment:
? Why this change?

Reply:
checkout@v2 seems to have some improvements - but reading through them it's not something the project needs at the moment - apologies for exceeding the scope on this piece. 
I've removed this change